<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Movement_Correction_Standalone</class>
 <widget class="QMainWindow" name="Movement_Correction_Standalone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>scan range</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="scan_range">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_step_size">
          <property name="text">
           <string>step size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="step_size">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.25</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.1</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushAnalyze">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Analyze</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="filter_list">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Savitzky_Golay</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low_Pass</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>savgol window</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="savgol_window">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>51</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>savgol_poly-order</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="savgol_poly">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="value">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushCorrect">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Correct</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>brightness</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushReconstruct">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reconstruct</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="brightness">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>999999</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_save_shifted_projections">
          <property name="text">
           <string>save shifted Projections</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushApplyVolume">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Apply to Volume</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushDoNotApply">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Do not apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>4008</width>
          <height>4008</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="plotfield">
         <attribute name="title">
          <string>Plot</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="PlotWidget" name="Graph">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Original Sinogram</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>3022</width>
               <height>2022</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QLabel" name="original_sino">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>3000</width>
                  <height>2000</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>3000</width>
                  <height>2000</height>
                 </size>
                </property>
                <property name="text">
                 <string>original_sino</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Corrected Sinogram</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>3022</width>
               <height>2022</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QLabel" name="corrected_sino">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>3000</width>
                  <height>2000</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>5000</width>
                  <height>5000</height>
                 </size>
                </property>
                <property name="text">
                 <string>corrected_sino</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Original Reconstruction</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_3">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>3022</width>
               <height>2022</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="0" column="0">
               <widget class="QLabel" name="original_reco">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>3000</width>
                  <height>2000</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>5000</width>
                  <height>5000</height>
                 </size>
                </property>
                <property name="text">
                 <string>original_reco</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Corrected Reconstruction</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_4">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>3022</width>
               <height>2022</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QLabel" name="corrected_reco">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>3000</width>
                  <height>2000</height>
                 </size>
                </property>
                <property name="text">
                 <string>corrected_reco</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>8</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
